.azure-pipelines/1esmain.yml

```diff
@@ -0,0 +1,90 @@
+# Trigger the build whenever `main` or `rel/*` is updated
+trigger: none
```

Comment:
Is this intentional to not have any triggers?

Reply:
Yes for now. Not having any triggers will just let us trigger it manually in ADO